src/main/java/com/loofah/graph/api/models/database/Category.java

```diff
@@ -5,19 +5,21 @@
 public class Category {
 
     @Id
-    private int id;
+    private String id;
     private String title;
+    private String description;
 
-    public Category(int id, String title) {
+    public Category(String id, String title, String description) {
         this.id = id;
         this.title = title;
+        this.description = description;
     }
 
-    public int getId() {
+    public String getId() {
```

Comment:
Whats the reason for changing the Id's from int to String?

Reply:
yeah you know what, i'm not sure why i did that. should id's be int? (just to check before i change it all back)